apps/tests/tests/e2e.test.ts

```diff
@@ -0,0 +1,158 @@
+import {
+  paginateGetResources,
+  ResourceGroupsTaggingAPIClient,
+  ResourceTagMapping,
+} from '@aws-sdk/client-resource-groups-tagging-api';
+import type { Rule, RuleCheckResult } from '@sls-mentor/core';
+import { CustomARN } from '@sls-mentor/core';
+import * as rules from '@sls-mentor/core/src/rules';
```

Comment:
you can `import { rules } from '@sls-mentor/core';` it is an array of all the rules